pkg/scheduler/actions/backfill/backfill.go

```diff
@@ -18,6 +18,8 @@ package backfill
 
 import (
 	"github.com/golang/glog"
+	"github.com/kubernetes-sigs/kube-batch/pkg/scheduler/conf"
```

Comment:
```go
sys pkg

third part pkg

kube-batch pkg
```

Reply:
To solve this kind of problem, we could use [goimports](http://goinbigdata.com/goimports-vs-gofmt/) to do format. How about it?

Reply:
that's ok to me :)

Reply:
the new commit in the master branch to use goimports brought lots of lines like the this:

![image](https://user-images.githubusercontent.com/252020/56836269-94bff000-682b-11e9-8f2a-9458963cc658.png)

here v1 is redundant. Why exactly?

Reply:
that's ok to me if generated by tools; please verify that.

Reply:
Seems to me more like a bug from the tools.

@hex108 made the change. Could you please verify these are generated on purpose?

Reply:
It is intended, https://github.com/golang/go/issues/30051

Reply:
will rebase nonetheless. my question remains.